style.css

```diff
@@ -0,0 +1,276 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap');
+*{
+    margin: 0;
+    padding: 0;
+    }
+    /* Navbar start */
+    nav{
+        display: flex;
+        height: 56px;
+        width: auto;
+        flex-direction: row;
+        align-items: center;
+        /* position: relative; */
+        /* background-color: black; */
+    }
+    .icon{
+        display: flex;
+        position: relative;
+        left: 23px;
+    }
+    .Micon{
+        /* white-space: 2px; */
+        width: 24px;
+        padding: 10px;
+        /* padding-top: 25px; */
+    }
+    .Micon img{
+        padding-top: 25px;
+    }
+    .Yicon img{
+        margin-top: 25px;
+    }
+    .search{
+        margin-left: 10%;
+        display: flex;
```

Comment:
Kindly one give any space before the class. Because it is not the professional way to do .
Also give the one space after the class